api/swagger-spec/v1.json

```diff
@@ -15011,6 +15015,95 @@
      }
     }
    },
+   "v1.Affinity": {
```

Comment:
are you sure you want to go straight to v1 on 1st cut vs through extensions?  Changing will be difficult then...